catchpoint-mixin/panels.libsonnet

```diff
@@ -9,6 +9,127 @@ local utils = commonlib.utils;
       local stat = g.panel.stat,
       local barGauge = g.panel.barGauge,
       local fieldOverride = g.panel.table.fieldOverride,
+      local alertList = g.panel.alertList,
+      topAvgLoadTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by test name',
+          targets=[t.topAvgLoadTimeTestName],
+          description='Average total time it took to load the webpage by test name, helping assess the responsiveness and speed of the monitored webpage.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
 
+      topAvgTotalLoadTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by node name',
+          targets=[t.topAvgTotalLoadTimeNodeName],
+          description='Average total time it took to load the webpage by test name, indicating the worst-case scenario for webpage loading performance.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by test name',
+          targets=[t.topAvgDocumentCompletionTimeTestName],
+          description='Average time taken for the browser to fully render the page after all resources are downloaded by test name, serving as a key indicator of end-user perceived load time.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by node name',
+          targets=[t.topAvgDocumentCompletionTimeNodeName],
+          description='Average time taken for the browser to fully render the page after all resources are downloaded by test name, highlighting the worst-case scenario for end-user perceived load time.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      bottomAvgRequestRatioTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Bottom average success request ratio by test name',
+          targets=[t.bottomAvgRequestSuccessRatioTestName],
+          description='Worst average succesful request ratio by test name.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withMax(1)
+        + g.panel.timeSeries.standardOptions.withMin(0)
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit'),
+
+      topAvgFailedRequestRatioNodeName:
```

Comment:
I think keeping this panel similar to the bottom average success request ratio by but my node name instead of test name is better for consistency. Other panel values can be updated similarly.